Content.Shared/Forensics/Components/ForensicScannerComponent.cs

```diff
@@ -1,43 +1,44 @@
 using System.Threading;
 using Robust.Shared.Audio;
+using Robust.Shared.GameStates;
 using Robust.Shared.Prototypes;
 using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
 
-namespace Content.Server.Forensics
+namespace Content.Shared.Forensics.Components
 {
-    [RegisterComponent]
+    [RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
     public sealed partial class ForensicScannerComponent : Component
     {
         public CancellationTokenSource? CancelToken;
 
         /// <summary>
         /// A list of fingerprint GUIDs that the forensic scanner found from the <see cref="ForensicsComponent"/> on an entity.
         /// </summary>
-        [ViewVariables(VVAccess.ReadOnly), DataField("fingerprints")]
+        [ViewVariables(VVAccess.ReadOnly), DataField("fingerprints"), AutoNetworkedField ]
```

Comment:
```suggestion
        [DataField, AutoNetworkedField]
```
no need for this to be readonly
same for the other datafields